membership/src/main/kotlin/net/corda/v5/membership/MemberInfo.kt

```diff
@@ -58,6 +60,7 @@ interface MemberInfo {
     val name: MemberX500Name
     val sessionInitiationKey: PublicKey
     val ledgerKeys: List<PublicKey>
+    val notaryKeys: List<PublicKey>
```

Comment:
Does this have to be exposed on the public API? I think notary keys will only be exposed publicly through the Notary Service. This could be better as an extension property within the layer that requires it (e.g. the ledger layer). For example, we have other non-public fields defined in `MemberInfoExtension.kt` in the corda runtime project.

Reply:
That'd be good to have as an extension then. Thanks 👍